### PR TITLE
Chat

### PR DESCRIPTION
**Pull request by danielPeng-17**
pr #1 of many

### CODE REVIEW

File: duos_ui/lib/screens/chat_page.dart [hunk 1]
```diff
@@ -1,201 +1,195 @@
...
-  const _MessageTile({
-    Key? key,
-    required this.message,
-  }) : super(key: key);
 
-  final String message;
+  void readLocal() {
+    // uid1 = authProvider.sub;
+    // uid2 = widget.arguments.peerUid;
+    uid1 = "PbsHaVsmcZQtspSJvAPlzgCPmP72";
```
Comment (KinglyYee) [inline duos_ui/lib/screens/chat_page.dart:52]: hardcoded value?

File: duos_ui/lib/screens/chat_page.dart [hunk 2]
```diff
@@ -1,201 +1,195 @@
...
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(
+          backgroundColor: Colors.transparent,
+          centerTitle: true,
+          title: Row(
+            children: const [
+              Padding(
+                padding: EdgeInsets.only(right: 12, left: 20),
+                child: Avatar.small(url: "https://picsum.photos/200/300"),
```
Comment (KinglyYee) [inline duos_ui/lib/screens/chat_page.dart:67]: another hardcoded value picture?

File: duos_ui/lib/screens/chat_page.dart (hunk 1 above)
Comment (danielPeng-17) [reply duos_ui/lib/screens/chat_page.dart:52]: for its hardcoded. will be changed once i get the contact page working

File: duos_ui/lib/screens/chat_page.dart (hunk 2 above)
Comment (danielPeng-17) [reply duos_ui/lib/screens/chat_page.dart:67]: will change once contact page is working

**Review by KinglyYee — APPROVED** [review]

**Pull request merged**